.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,20 @@
 ## 🗒️ Description
 <!-- Brief description of the changes introduced by this PR -->
+<!-- Don't submit this PR if it could expose a mainnet bug, see [SECURITY.md](/ethereum/execution-spec-tests/SECURITY.md) -->
+<!-- Consider running the following fast checks before pushing changes to avoid unnecessary CI fails, see also "[Enabling Pre-commit Checks](https://eest.ethereum.org/main/dev/precommit/)"-->
+<!-- uvx --with=tox-uv tox -e lint,typecheck,spellcheck,markdownlint -->
 
-## 🔗 Related Issues
-<!-- Reference any related issues using the GitHub issue number (e.g., Fixes #123) -->
+## 🔗 Related Issues or PRs
+<!-- Reference any related issues using the GitHub issue number (e.g., Fixes #123). Default is N/A. -->
+N/A.
 
 ## ✅ Checklist
+<!-- Please check off all required items. For those that don't apply remove them accordingly. -->
 
-- [ ] All: Set appropriate labels for the changes.
-- [ ] All: Considered squashing commits to improve commit history.
-- [ ] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
+- [ ] All: Set appropriate labels for the changes (only maintainers can apply labels).
```

Comment:
```suggestion
- [ ] All: Ran fast `tox` checks to avoid unnecessary CI fails, see also [Code Standards](https://eest.ethereum.org/main/getting_started/code_standards/) and [Enabling Pre-commit Checks](https://eest.ethereum.org/main/dev/precommit/):
    ```console
    uvx --with=tox-uv tox -e lint,typecheck,spellcheck,markdownlint
    ```
```

Reply:
This comment will often come too late: When a Contributor sees this page, they'll already have pushed their changes. But still, if they get a CI fail, they will immediately see here how they can remedy fails in fast checks and get a link to docs for other fails.